healthScore/healthScore/templates/view_history.html

```diff
@@ -88,22 +88,23 @@ <h2 class="text-center">Your Health History</h2>
     <div class="row">
         <div class="col-lg-12">
             <h4>Filter Your Health History</h4>
-            <form method="GET" action="{% url 'view_health_history' %}" class="form-inline">
+            <form method="POST" action="{% url 'view_health_history' %}" class="form-inline">
+                {% csrf_token %}
                 <div class="form-group mx-sm-3 mb-2">
                     <label for="appointment_name" class="sr-only">Reason for Visit:</label>
-                        <input type="text" class="form-control" id="appointment_name" name="appointment_name" placeholder="Reason for Visit" value="{{ request.GET.appointment_name }}">
```

Comment:
These value="{{}}" statements keep the values the user is searching for in the filtering search bars. They might not know what they searched for if it doesn't stay in there.